File: client/src/dtos/blogs/blogListDto.ts
export interface BlogListDto {
    id: number;
    title: string;
    categoryId: number;
    categoryName: string;
    categoryIsDeleted: boolean;
    createdAt: string;
    imageUri: string;
    authorFullName: string;
    authorId: number;
    authorImageUri: string;
    isDeleted: boolean;
    isHidden: boolean;
}

File: server/dto/blog/blogListDto.go
package blog_dto

type BlogListDto struct {
	Id             uint   `json:"id"`
	Title          string `json:"title"`
	CategoryId     uint   `json:"categoryId"`
	CategoryName   string `json:"categoryName"`
	CreatedAt      string `json:"createdAt"`
	ImageUri       string `json:"imageUri"`
	AuthorFullName string `json:"authorFullName"`
	AuthorId       uint   `json:"authorId"`
	IsDeleted      bool   `json:"isDeleted"`
	IsHidden       bool   `json:"isHidden"`
}
